source/parser/comptime.civet

```diff
@@ -139,10 +139,31 @@ function serialize(value: ???): string
             when Array::
               `[${(val as ???[]).map(recurse).join ","}]`
             when Object::
-              "{" + (
-                for own key, value in val as {[key: string]: ???}
-                  `${JSON.stringify key}:${recurse value}`
-              ).join(",") + "}"
+              // The string representing the object's own enumerable, writable properties
+              objStr .= '{'
+              // The string representing other property descriptors, if applicable, not including the outermost braces
+              descStr .= ''
+              descriptors := Object.getOwnPropertyDescriptors(val) as Record<string | symbol, PropertyDescriptor>
+              for prop of (Object.getOwnPropertyNames(val) as (string|symbol)[]) ++ Object.getOwnPropertySymbols(val)
+                desc := descriptors[prop]
+                if desc.enumerable and desc.configurable and desc.writable
```

Comment:
Do you need to check for `get` and `set` for custom getters and setters? Or I guess that makes an accessor descriptor which won't have `writable` set?

Reply:
A descriptor will never have both `writable` and `get` defined, and `undefined` is falsy.